src/backend/marsha/core/tests/test_api_liveregistration.py

```diff
@@ -3742,36 +3748,10 @@ def test_api_liveregistration_put_username_public_already_exists(
         self.assertEqual(response.status_code, 409)
         self.assertEqual(
             response.json(),
-            {"display_name": "User with that username already exists!"},
+            {"display_name": "User with that display_name already exists!"},
         )
 
```

Comment:
suggestion test_api_liveregistration_put_username_public_no_anonymous to modify the comment  """Field anonymous_id is mandatory.""" =>  """Field anonymous_id is mandatory when token is not lti"""